tests/src/common.hh

```diff
@@ -121,3 +128,66 @@ template <typename NonLinearOperator>
       << "The Jacobian of calculateMatrix is not the Jacobian of calculateVector." << messageIfFailed;
   return t;
 }
+
+template <typename NonLinearOperator, typename FiniteElement, typename GridView>
+[[nodiscard]] auto checkCauchyStressOf2DElement(NonLinearOperator& nonLinearOperator, FiniteElement& fe,
+                                                GridView& gridView,
```

Comment:
I don't thnik yoi have to pass `gridView`. You can access it via `fe.localView().globalBasis().gridView()` i suppose. but there may be a better way. Any the functino parameters should be only the old two. But maybe I'm missing something?